IFC4x3/PropertyEnumerations/p/PEnum_ProjectType.xml

```diff
@@ -1,9 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_ProjectType" Name="PEnum_ProjectType" UniqueId="c0481c6f-beda-4e2e-a51e-16e1132936d1">
+	<Documentation>Enumeration describing the typing of a project</Documentation>
 	<Constants>
 		<DocPropertyConstant xsi:nil="true" href="NEWBUILD_3_S218crb2exItJY2XPrej" />
 		<DocPropertyConstant xsi:nil="true" href="MODIFICATION_37GiebKMXDm9FwpugQb_Xg" />
 		<DocPropertyConstant xsi:nil="true" href="REPAIR_3NBX9auO1EJQ8qly_XHl7f" />
+		<DocPropertyConstant xsi:nil="true" href="OPERATIONMAINTENANCE_2ouZNUsh55rxlrJJSTNVly" />
```

Comment:
I love this. Finally someone had the guts to formalize it! :) People are talking about this in all their presentations but nobody knew that iIFC does not actually support it?

Reply:
indeed it a primary property on most CDEs when you set your project info, but we neve had a Pset_Project to put it somewhere!